templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,50 @@
+    <h3>Watch successfully boots into the AsteroidOS user interface, but immediately reboots</h3>
+    This is a known behavior when the battery charge is very low. The watch usually recovers from this condition by simply boot cycling for some time. All watches also charge in fastboot or recovery mode. Letting the watch dwell on the fastboot or recovery menu screen for some time should fix the problem even faster.
 </div>
-
+<h2>Flashing problems</h2>
 <div class="callout">
-    <h4>Flash was successful with fastboot throwing no errors, but watch does not boot</h4>
-    The flashing process can silently fail due to an unstable connection. Make sure you cleaned the contacts on the cradle and the watch. Redo the flashing process up to four times. On watches with especially bad connection, it can help to fixated the watch to the cradle using ductape or rubber bands.
-</div>
-
-<div class="callout">
-    <h4>My fastboot command displays "invalid sparse file format at header magic" when starting to flash the device</h4>
-    This is not a fatal error and can safely be ignored.
-</div>
-
-<div class="callout">
-    <h4>My fastboot command crashes or hangs at "invalid sparse file format at header magi" (with a missing "c" in magic)</h4>
-    This error occurs when using deprecated fastboot and ADB commands on Windows systems. If you already had those commands installed and skipped downloading our <a href="https://release.asteroidos.org/tools/adb_1_0_39+fastboot+ext2simg.zip">supplied zip file</a>, please install those commands instead.
-</div>
-
-<div class="callout">
-    <h4>My Asus Zenwatch 2 flashed successfully but does not boot</h4>
-    There are two versions of the Asus Zenwatch 2, codenamed Sparrow and Wren. Their images are not interchangeable. Please ensure you are using the right files.
+    <h3>Watch is not detected and all fastboot commands fail</h3>
+    <ul>
+      <li><h4>When using Linux,</h4> try prepending <code>sudo</code> to the commands to execute them with root privileges. If that works, you are missing UDEV rules for Android devices that would allow your user to execute the commands. Many package managers provide the <code>android-udev-rules</code>, which avoids the need to manually set up these UDEV rules.</li>
```

Comment:
Maybe also provide a direct link in case a package isn't available? 
(https://github.com/M0Rf30/android-udev-rules)

Reply:
Greta idea!

Reply:
is this okay?
`If the package is not available on your system, you can follow <a href="https://github.com/M0Rf30/android-udev-rules">the instructions at the official android-udev-rules repository</a>.`

Reply:
Sounds good :+1: